content/en/blog/2021/etcd-deployments-on-AWS.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Deployment of etcd on AWS EC2
+author: Caleb Miles, Somoshree Datta
```

Comment:
As with @spzala's comment https://github.com/etcd-io/website/pull/469#discussion_r714322007, probably we should only have Caleb Mile's name here as an author. Not to discount the work you've done on this @Somoshree, but this work is more adaptation than authorship.

I'm not sure we need any authorship in this line as we're already citing Caleb in the note at the bottom of the page.